app/pages/items/description/contextCreator.js

```diff
@@ -4,5 +4,4 @@ import { baseUrl } from '../../../config';
 export const getContext = ({ orderId }) => ({
```

Comment:
🙄  can we change the folder name from `items` to `sections` please

Reply:
nooooooooooooooooooooooooooo........